### PR TITLE
created Car class

### PR DESCRIPTION
**Pull request by anahittum**
created Car class

### CODE REVIEW

File: src/homeworks/week2/cars/Car.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package homeworks.week2.cars;
+
+public class Car {
```
Comment (gevorgyanarman) [inline src/homeworks/week2/cars/Car.java:3]: Create a method which takes cars and returns list of cars manufactured after 2010.

The requirement is not done

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

File: src/homeworks/week2/cars/Car.java (hunk 1 above)
Comment (anahittum) [reply src/homeworks/week2/cars/Car.java:3]: Please take a look at printSelectedCars() method.

Comment (anahittum) [reply src/homeworks/week2/cars/Car.java:3]: Modified.

File: src/homeworks/week2/cars/Car.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public String getBrand() {
+        return brand;
+    }
+
+    @Override
+    public String toString() {
+        return brand + " " + year;
+    }
+
+    private static void printSelectedCars(int year, Car[] cars) {
```
Comment (gevorgyanarman) [inline src/homeworks/week2/cars/Car.java:25]: Still, the code does not meet the requirement. You must create a method that takes a Car[] and returns a new Car array.

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**